tests/toolbox_image.sh

```diff
@@ -159,6 +228,26 @@ test_system_update_entrypoint()
     test -x "${rootfs_dir}${SYSTEM_UPDATE_ENTRYPOINT}"
 }
 
+test_execute_resize_partition_shrink_spool_ok()
+{
+    partition_table_file="/tmp/partition_table"
```

Comment:
why not use chroot ${rootfs_dir} mktemp? (if you can't store it in a variable instead)
remember however that you also need to clean it up when things fail!

Reply:
Putting it in a variable was what i had before we decided to do it with the toolbox tools in the chroot -ed environment.    
I don't think it is possible, normally you would 'echo partition_table | fdisk image_file' i don't see how this can be done in chroot.

Reply:
i'm not seeing the cleanup of the file however now.
Maybe pre-generate a in setup() and clean it up in teardown().